src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IForEachLoopStatement.cs

```diff
@@ -71,7 +74,9 @@ static void Main(string[] args)
 }
 ";
             string expectedOperationTree = @"
-IForEachLoopStatement (Iteration variable: System.String item) (LoopKind.ForEach) (OperationKind.LoopStatement) (Syntax: 'foreach (st ... }')
+IForEachLoopStatement (LoopKind.ForEach) (OperationKind.LoopStatement) (Syntax: 'foreach (st ... }')
+  Locals: Local_1: System.String item
+  LoopControlVariable: ILocalReferenceExpression: item (IsDeclaration: True) (OperationKind.LocalReferenceExpression, Type: System.String, Constant: null) (Syntax: 'foreach (st ... }')
   Collection: IConversionExpression (Implicit, TryCast: False, Unchecked) (OperationKind.ConversionExpression, Type: System.Collections.Generic.List<System.String>) (Syntax: 'list')
```

Comment:
>IConversionExpression (Implicit, TryCast: False, Unchecked) (OperationKind.ConversionExpression, Type: System.Collections.Generic.List<System.String>) [](start = 14, length = 150)

Do we really need this implicit identity conversion?

Reply:
For both languages, we should examine what implicit conversions are exposed on top of collection and whether they are expected. 

---
In reply to: [137703658](https://github.com/dotnet/roslyn/pull/21791#discussion_r137703658) [](ancestors = 137703658)

Reply:
Filed https://github.com/dotnet/roslyn/issues/22031